controllers/storagecluster/cephfilesystem.go

```diff
@@ -121,11 +122,79 @@ func (obj *ocsCephFilesystems) ensureCreated(r *StorageClusterReconciler, instan
 				return reconcile.Result{}, err
 			}
 		}
+		// create default csi subvolumegroup for the filesystem
+		// skip for the ocs provider mode
+		if !instance.Spec.AllowRemoteStorageConsumers {
+			err = r.createDefaultSubvolumeGroup(cephFilesystem.Name, cephFilesystem.Namespace, cephFilesystem.ObjectMeta.OwnerReferences)
+			if err != nil {
+				return reconcile.Result{}, err
+			}
+		}
 	}
 
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) createDefaultSubvolumeGroup(filesystemName, filesystemNamespace string, ownerReferences []metav1.OwnerReference) error {
+	// TODO: After fix of rook issue https://github.com/rook/rook/issues/13220 add svg name spec
+
+	existingsvg := &cephv1.CephFilesystemSubVolumeGroup{}
+	err := r.Client.Get(r.ctx, types.NamespacedName{Name: "csi", Namespace: filesystemNamespace}, existingsvg)
```

Comment:
Can we name this SVG as a filesystem name instead of a generic name `csi`? Also lets create a constant for this as this name is used at multiple places.

Reply:
Updated